qp_klp/tests/data/process_all_fastq_files.sh

```diff
@@ -0,0 +1,265 @@
+#!/bin/bash -l
```

Comment:
Why is this file now needed in the tests vs. just leaving it in the templates?

Reply:
it's _much_ less time-consuming to simply load the expected value from a file and compare it with the observed results rather than keep trying to make point changes to a list of strings that are often split to placate flake8. We really do modify that batch script often, at least currently.

Reply:
however, I thought we were not adding tests that compared this file in a tests ... no?

Reply:
we're not adding new tests but this was an existing test. I could replace the test for one that simply checks for the existence of a job-script in the output and maybe checks the first few lines, but that seems a step backwards.